tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+   "source": [
+    "The functions `dir, help` is helpful for investigating through sympy modules, functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39a523f0-e504-488c-9ae9-74dc6f56d202",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Try `dir`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07dd13cc-dad5-4a3a-b0dd-1fbfedbde0d2",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "dir(sympy)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b47b39d-b8df-4248-9ce7-935a81ccc668",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Try `help`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94f7d16d-8cfd-440d-99f3-d82bf8be9ac9",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "help(sin)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5723241-fe0f-4d89-b45b-c0a5f29f575e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "sin?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25a573ae-00bf-4c51-85ba-adbe4f083651",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Browse SymPy documentation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb9af4d9-2243-4707-9b37-bc5b0eacf293",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You may also read the SymPy documentation\n",
+    "\n",
+    "https://docs.sympy.org/latest/index.html\n",
+    "\n",
+    "for more formatted help."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70dae286-468d-4f6b-a929-cdd957b563ea",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Symbolic Computation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e31ae6c2-c08f-4cd9-95dc-6cbc0b060e18",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Integers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7279349-50e6-4747-93b0-1fb5c6764de7",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "In many computer languages you may have learned through education (C, JAVA, ...), integers are not infinite and prone to overflow errors.\n",
```

Comment:
Our tutorial prerequisites are for the attendees to already know Python (but not any other language).